internal/rest/types/cluster.go

```diff
@@ -48,3 +48,8 @@ const (
 	// MemberNeedsUpgrade should be the MemberStatus if the system needs to receive a schema upgrade to be compatible with other cluster members.
 	MemberNeedsUpgrade MemberStatus = "NEEDS UPGRADE"
 )
+
+const (
+	// ClusterCertificate represents the name of the cluster certificate used by the core API.
+	ClusterCertificate string = "cluster"
```

Comment:
This should probably be called `ClusterCertificateName` to avoid any confusion.

Reply:
Fixed